src/dr_action.py

```diff
@@ -130,21 +130,20 @@ def add_dump_ctx(self, dump_ctx):
         self.dump_ctx = dump_ctx
 
     def dump_str(self):
+        if simple_output:
+            if ( (_srd(self.data, "id")) in self.dump_ctx.modify_hdr.keys()):
+                out_str = self.dump_ctx.modify_hdr[(_srd(self.data, "id"))].lstrip(',')
+                return "MODIFY_HDR(hdr(%s))" % (out_str)
+
```

Comment:
Line 137: add:
       else return "MODIFY_HDR"

Reply:
OK

Reply:
i think  "else" is for if ( (_srd(self.data, "id")) in self.dump_ctx.modify_hdr.keys()):  . IF we do not find the id in key, then simply print "MODIFY_HDR".

Reply:
right, wrong indent. fixed.